config/initializers/_qiniu_sdk.rb

```diff
@@ -0,0 +1 @@
+Qiniu.establish_connection! Rails.application.secrets[:qiniu].symbolize_keys
```

Comment:
我应该是删了原来那个的啊，回去看看
2016年5月23日 下午12:34，"Rain Chen" <notifications@github.com>写道：

> In config/initializers/_qiniu_sdk.rb
> <https://github.com/beansmile/jianggaowang/pull/240#discussion_r64168681>:
>
> > @@ -0,0 +1 @@
> > +Qiniu.establish_connection! Rails.application.secrets[:qiniu].symbolize_keys
>
> 已经有一份“qiniu_sdk.rb”， 不能加入这里？
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/beansmile/jianggaowang/pull/240/files/7903f2af92df3a910ce7d3d4978f15e827cf73b8#r64168681>
>